### PR TITLE
French translation (SabakiHQ)

### PR DESCRIPTION
**Pull request by romton843**

### CODE REVIEW

**Comment by romton843** [comment]
The French translation is complete.
I couldn't load it with Sabaki developer.
I was able to load the other languages but not mine and I don't know why exactly...
Can someone help me out so that I can review it.
Best regards.
Romain

File: src/fr.i18n.js [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+    'Enclosure Opening': 'Fuseki Verrou de Coin',
+    'Kobayashi Opening': 'Fuseki Kobayashi',
+    'Small Chinese Opening': 'Mini Fuseki Chinois',
+    'Micro Chinese Opening': 'Micro Fuseki Chinois',
+    'Sanrensei Opening': 'Fuseki Sanrensei',
+    'Nirensei Opening': 'Fuseki Nirensei',
+    'Shūsaku Opening': 'Fuseki Shūsaku',
+    '3-3 Point': 'Point 3-3',
+    'Low Approach': 'Approche Basse',
+    'High Approach': 'Approche Haute,
```
Comment (yishn) [inline src/fr.i18n.js:512]: Missing closing quote `'` here.

File: src/fr.i18n.js [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+    'Square': 'Carré',
+    'Throwing Star': 'Étoile de Ninja,
+    'Parallelogram': 'Parallélogramme',
+    'Dog’s Head': 'Nez de Chien',
+    'Horse’s Head': 'Nez de Cheval',
+    'Attachment': 'Contact',
+    'One-Point Jump': 'Saut d’une Intersection',
+    'Big Bulge': 'Grand Renflement',
+    'Small Knight': 'Saut de Cheval',
+    'Two-Point Jump': 'Saut de Deux Intersections,
```
Comment (yishn) [inline src/fr.i18n.js:538]: Missing closing quote `'` here.

File: src/fr.i18n.js [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+    'Tiger’s Mouth': 'Gueule de Tigre',
+    'Empty Triangle': 'Angle Vide',
+    'Turn': 'Coude',
+    'Stretch': 'Étirement',
+    'Diagonal': 'Diagonale',
+    'Wedge': 'Encastrement',
+    'Hane': 'Hane',
+    'Cut': 'Coupe',
+    'Square': 'Carré',
+    'Throwing Star': 'Étoile de Ninja,
```
Comment (yishn) [inline src/fr.i18n.js:530]: Missing closing quote `'` here.

File: src/fr.i18n.js (hunk 1 above)
Comment (romton843) [reply src/fr.i18n.js]: Fixed

File: src/fr.i18n.js (hunk 2 above)
Comment (romton843) [reply src/fr.i18n.js]: Fixed

File: src/fr.i18n.js (hunk 3 above)
Comment (romton843) [reply src/fr.i18n.js]: Fixed

**Pull request merged**